tests/formattedcode/test_writers.py

```diff
@@ -412,3 +412,11 @@ def test_spdx_rdf_tree():
     result = run_scan_click(['--format', 'spdx-rdf', test_dir, result_file])
     assert result.exit_code == 0
     check_rdf_scan(expected_file, result_file)
+
+def test_csv():
```

Comment:
I think there are many other tests to carry over from https://github.com/nexB/scancode-toolkit/blob/4a1d1d3df5b2121b6a42d88c5796d48cc59f602e/etc/scripts/test_json2csv.py right?